2-mandatory/4-people-i-know.js

```diff
@@ -394,6 +403,14 @@ This time, I only want the full names ("<firstname> <lastname>") of my friends w
 */
 
 let friendsWhoAreColleaguesOfStacie = [];
+for (let myFriend of friends) {
+  for (let colleague of myFriend.colleagues)
```

Comment:
Even if it will work this way, it's recommended that you use `{}` for all `for` loops